module/zfs/zfs_ioctl.c

```diff
@@ -4605,6 +4623,29 @@ zfs_ioc_userspace_upgrade(zfs_cmd_t *zc)
 	return (error);
 }
 
+/*
+ * inputs:
+ * zc_name		name of filesystem
+ *
+ * outputs:
+ * none
+ */
+static int
+zfs_ioc_userobjspace_upgrade(zfs_cmd_t *zc)
+{
+	int err;
+	zfs_sb_t *zsb;
+
+	err = zfs_sb_hold(zc->zc_name, FTAG, &zsb, B_FALSE);
+	if (err != 0)
```

Comment:
`get_zfs_sb()` should be used here since `zfs_sb_hold()` can result in a super block being created.  It would be better to model this after `zfs_ioc_userspace_upgrade()` which just takes a hold on the objset when there's no existing super block.